app/src/main/java/com/tari/android/wallet/application/WalletManager.kt

```diff
@@ -145,7 +145,8 @@ internal class WalletManager(
             Constants.Wallet.walletDBName,
             walletFilesDirPath,
             Constants.Wallet.discoveryTimeoutSec,
-            Constants.Wallet.storeAndForwardMessageDurationSec
+            Constants.Wallet.storeAndForwardMessageDurationSec,
+            "weatherwax"
```

Comment:
We keep network names in Network enum
```suggestion
            "Network.WEATHERWAX.uriComponent"
```

Reply:
While I could do this, this would need to change further when the UI for network selection is done. So would end up being something like `Constants.Wallet.currentNetwork.urlComponent`.

Thoughts?